README.md

```diff
@@ -1,3 +1,7 @@
+# This is a fork of the original app
```

Comment:
Sorry, I didn't realise that the pull request would update automatically to tip of the branch.